MariaDB on-prem_Fixed UAT Bugs (#260)
IBM/universal-connectors: filter-plugin/logstash-filter-mariadb-guardium/README.md

Filebeat provides a solution which can be implemented to ignore such logs, below are the steps
1. Open filebeat.yml in notepad or any OS specific text editor.
2. Search "filebeat.inputs" property.
3. Under the "filebeat.inputs" property, uncomment the exclude_lines property and place the query user wants to ignore at filebeat level

> 4. Different third party tool may generate different system queries, so based on observation user need to modify queries in 'exclude line' of filebeat configuration.

Reviewer: ```suggestion
4. Different third party tools may generate different system queries, so modify queries in the 'exclude line' of your Filebeat configuration as needed.
```

 
## Before you begin
* You must have permission for the S-Tap Management role. The admin user includes this role by default.
* Download the [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in.
* Download the plugin filter configuration file [MariaDB.conf](MariaDB.conf).

## Procedure
1. On the collector, go to Setup > Tools and Views > Configure Universal Connector.
2. Enable the connector if it is disabled before uploading the UC plug-in.	

> 3. Click Upload File and select the offline [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in. After it is uploaded, click OK.

Reviewer: ```suggestion
3. Click ```Upload File``` and select the offline [logstash-filter-mariadb_guardium_filter.zip](MariaDBOverFilebeatPackage/logstash-filter-mariadb_guardium_filter.zip) plug-in. After it is uploaded, click ```OK```.
```
